cfgov/v1/atomic_elements/molecules.py

```diff
@@ -83,6 +83,15 @@ class AbstractHero(blocks.StructBlock):
             "</ul>"
         ),
     )
+
+    heading_with_line_break = blocks.CharBlock(
```

Comment:
This may be too pedantic to matter, but this isn't a heading with a line break, right? It's just a continuation of the heading and then the line break gets added adjacent to it in the template. The only problem with this might be if in the future the `<br />` in the template were to be removed for whatever reason and then we have this named `heading_with_line_break` when it doesn't actually contain a line break. @willbarton would it make any sense to be adding the `<br />` to the text entered here before it ends up in the template? Alternatively I think it would be more accurate as something like `heading_continued` or something, which better conveyed that this is just a continuation of the heading text (which likely would be separated by a line break, but we'll leave that up to the template where it's used).

Reply:
>  @willbarton would it make any sense to be adding the <br /> to the text entered here before it ends up in the template?

That feels anti-patternish to me, since adding HTML around values is what the templates are for?

Reply:
> That feels anti-patternish to me, since adding HTML around values is what the templates are for?

Yeah, but we do add markup in the middleware in some cases, right? For example, with links that get the external link icons added? I wasn't sure if this perhaps fit into a similar convention. It does seem better to just have the markup in a template though, but then again the name `heading_with_line_break` would not be accurate, because it doesn't actually contain the line break.

Reply:
changed in https://github.com/cfpb/consumerfinance.gov/pull/8191/commits/374073de5799829c008bc8e4281eddfd4d57220a